### PR TITLE
Add cross domain iframe strategy

### PR DESCRIPTION
**Pull request by paulvarache**
Related to https://trello.com/c/WfiZjOUQ/264-3-unified-login

### CODE REVIEW

File: xdAuthIframe.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<html>
+    <head></head>
+    <body>
+        <script>
+            var MESSAGE_NAMESPACE = 'kw-xd-auth',
+                authorizedOrigins = [
+                    /^http(s)?:\/\/(.*)(\.)?kano.me$/,
```
Comment (arifshanji) [inline xdAuthIframe.html:7]: We have kano.zendesk.com hitting our API.  It should be in this list.

**Comment by rcocetta** [comment]
You need to bump the package.json to release this

**Comment by sylar** [comment]
can't we implement a more elegant method? like sso?

**Review by arifshanji — CHANGES_REQUESTED** [review]

File: xdAuthIframe.html (hunk 1 above)
Comment (paulvarache) [reply xdAuthIframe.html:7]: This origin refer to the frontend apps calling the iframe. Something calling the API won't be affected

**Comment by paulvarache** [comment]
I'd be happy to do so. If you know a solution to share localStorage data between domains, please send me more info about it and we'll re-architecture the thing

**Review by arifshanji — APPROVED** [review]

**Comment by sylar** [comment]
we could've used a strategy of sso of passing a query string, verifying it and singing in the user  accordingly (writing to localstorage/cookie) and avoinding hidden iframes ... anyway, is redundant now.

**Pull request merged**